obspy/imaging/scripts/scan.py

```diff
@@ -192,6 +196,373 @@ def load_npz(file_, data_dict, samp_int_dict):
         npz_dict.close()
 
 
+def _seconds_to_days(seconds):
+    return seconds / (24 * 3600)
+
+
+class Scanner(object):
+    """
+    """
+    def __init__(self, format=None, verbose=False, quiet=True, recursive=True,
+                 ignore_links=False, starttime=None, endtime=None,
+                 seed_ids=None):
+        self.format = format
+        self.verbose = verbose
+        self.quiet = quiet
+        self.recursive = recursive
+        self.ignore_links = ignore_links
+        if starttime is None:
+            self.starttime = None
+        else:
+            self.starttime = date2num(starttime.datetime)
```

Comment:
we have `UTCDateTime.matplotlib_date` now but I understand that this is just a copy/paste job.